operatorcourier/push.py

```diff
@@ -21,14 +21,20 @@ class PushCmd():
     def __init__(self):
         pass
 
-    def push(self, bundle_dir, namespace, repository, release, auth_token):
+    def push(self, bundle_dir, namespace, repository, release, auth_token,
+             quay_host, verify_host):
         """Push takes a bundle and pushes it to the specified app registry repository.
 
         :param bundle_dir: Path to generated local directory that contains the bundle.
         :param namespace: Namespace that contains the repository for the application.
         :param repository: Repository name of the application described by the bundle.
         :param release: Release version of the bundle.
         :param auth_token: Authentication token used to push to Quay.io.
+        :param quay_host: Can be 'quay.io' or your private instance hostname.
+        :param verify_host: Either a boolean, in which case it controls whether we verify
+                            the server's TLS certificate, or a string, in which case
+                            it must be a path to a CA bundle to use.
+                            Defaults to ``True``.
         """
         logger.info('Generating 64 bit bundle and pushing to app registry.')
         filterOutFiles(bundle_dir, BLACK_LIST)
```

Comment:
Just below this, don't you also need to update the call to `_push_to_registry()` to actually pass the new parameters along?